packages/backend-test-utils/api-report.md

```diff
@@ -272,6 +283,29 @@ export namespace mockServices {
       ) => ServiceMock<RootConfigService>;
   }
   // (undocumented)
+  export function rootEventAuditor(
+    options?: rootEventAuditor.Options,
```

Comment:
Is there a use case for creating an _actual_ auditor with options? It would make sense to me to leave out this callback function form entirely and keep it simple and similar to the non-root auditor above.

Reply:
This was implemented to keep `rootAuditor` aligned with `rootLogger`, allowing the user to specify a minimum logging level.